lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -26,7 +26,8 @@ module Framework
           tracer: Datadog.tracer,
           debug: false,
           trace_agent_hostname: Datadog::Writer::HOSTNAME,
-          trace_agent_port: Datadog::Writer::PORT
+          trace_agent_port: Datadog::Writer::PORT,
+          env: ::Rails.env
```

Comment:
> The Rails convention is not to have your staging and production environments use the same environment name.

Hunh. Today I Learned ...

OK. I can set aside my philosophical differences with what turns out to be convention and make `nil` my default.  Thanks!